app/src/main/res/values-de/strings.xml

```diff
@@ -6,6 +6,7 @@
     <string name="option_show_date">Datum anzeigen</string>
     <string name="option_use_emoji">Emoji anzeigen</string>
     <string name="option_price_group">Preisgruppe</string>
+    <string name="option_filter_deals">Angebote ignorieren</string>
```

Comment:
Maybe "Werbung und Angebote ignorieren" is clearer? And in a potential redesign I would incloude an optional short description for every setting.